connectors/sources/confluence.py

```diff
@@ -37,12 +37,12 @@
 SPACE_QUERY = "limit=100"
 ATTACHMENT_QUERY = "limit=100&expand=version"
 CONTENT_QUERY = (
-    "limit=100&expand=children.attachment,history.lastUpdated,body.storage,space"
+    "limit=50&expand=children.attachment,history.lastUpdated,body.storage,space"
 )
 
 URLS = {
     SPACE: "rest/api/space?{api_query}",
-    CONTENT: "rest/api/content?{api_query}",
+    CONTENT: "rest/api/content/search?{api_query}",
```

Comment:
What about this change too?

Reply:
The search endpoint allows us to filter the contents based on spaces.